aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -541,7 +541,7 @@ The following services are registered during host initialization:
 * [Environment](xref:fundamentals/environments) (<xref:Microsoft.Extensions.Hosting.IHostingEnvironment>)
 * <xref:Microsoft.Extensions.Hosting.HostBuilderContext>
 * [Configuration](xref:fundamentals/configuration/index) (<xref:Microsoft.Extensions.Configuration.IConfiguration>)
-* <xref:Microsoft.Extensions.Hosting.IApplicationLifetime> (`Microsoft.Extensions.Hosting.Internal.ApplicationLifetime`)
+* <xref:Microsoft.Extensions.Hosting.IHostApplicationLifetime> (`Microsoft.Extensions.Hosting.Internal.ApplicationLifetime`)
```

Comment:
From this point down in the topic ... references to `IApplicationLifetime` past Line 484 ... revert the changes. The API change to `IHostApplicationLifetime` is for >=3.0, and everything from Line 484 down is <3.0 content.

https://github.com/aspnet/Announcements/issues/344